index.rst

```diff
@@ -1,61 +1,106 @@
 
-   Feel free to delete this instructional comment.
+.. _NIST SP 800-52 Rev. 2: https://csrc.nist.gov/publications/detail/sp/800-52/rev-2/final
 
-:tocdepth: 1
+The client may authenticate to the server inside the TLS-protected request via a mechanism other than mutual certificate authentication.
+Client certificate authentication is not required.
+However, the client must verify the server certificate as described in the NIST guidance.
 
-.. Please do not modify tocdepth; will be fixed when a new Sphinx theme is shipped.
+The one exception from the NIST SP 800-52 Rev. 2 guidelines is that FIPS 140 validation of the cryptographic modules used by the client and server will not be required.
+However, both the client and server must use **either** a FIPS-140-validated cryptographic module **or** a cryptographic module from a widely-used and generally-respected cryptographic implementation of TLS that receives active security support, such as OpenSSL_.
+This exception has been made because formal FIPS 140 validation is often not available for commercial and financial reasons unrelated to the quality of the cryptographic implementation, so this requirement could otherwise unnecessarily constrain the choice of software without a meaningful improvement in security for this application.
 
-.. sectnum::
+.. _OpenSSL: https://www.openssl.org/
+
+Data transfer from the summit facility to the base facility **may** use TLS encryption as described above if, for instance, a direct TCP connection from the summit to the USDF is deemed the best transfer strategy.
+However, if there is a separate transfer from the summit facility to the base facility, it may forgo encryption if that encryption could interfere with the requirements for Prompt Processing (in particular, the 60 second bound on elapsed time from shutter close to alert production).
+This is permissible because the connection from the summit facility to the base facility is over dedicated private fiber and the networking equipment at both ends is under the control of Vera C. Rubin Observatory.
+
+Communication from the :abbr:`DAQ (Data Acquisition System)` on the telescope to its client need not be (and is not) encrypted since this communication happens entirely within the summit facility.
+Only Rubin Observatory staff will have access to the summit facility or any of the systems running there.
+
+Project impact
+--------------
+
+Data transfer has previously been tested using bbcp_, a data transfer program for scientific data that is optimized for transfer speed.
+It encrypts the control channel (via SSH_) but not the data channel, and therefore does not meet this requirement.
+Data transfer will have to be reimplemented using TLS.
+
+.. _bbcp: https://www.slac.stanford.edu/~abh/bbcp/
+.. _SSH: https://en.wikipedia.org/wiki/Ssh_(Secure_Shell)
+
+To obtain sufficient transfer speeds over TLS, it is often necessary to chunk the data being transferred and transfer multiple chunks in parallel.
```

Comment:
Let's not go into this.  We have 200+ CCD images to transfer; we can divide them up across machines and connections however necessary to maximize throughput.

Reply:
Oh, good to know. Dropped.